addons/binding/org.openhab.binding.regoheatpump/src/main/java/org/openhab/binding/regoheatpump/internal/rego6xx/Rego6xxProtocolException.java

```diff
@@ -0,0 +1,15 @@
+package org.openhab.binding.regoheatpump.internal.rego6xx;
```

Comment:
Hm, I found out why: Because it is considered prio 3.
@svilenvul Did we do that by intention? I would actually hope that a missing header breaks the build - and where we have exceptions because we need different headers, it would be nice if we could deactivate this check in the pom of the appropriate bundle.